openpoiservice/server/db_import/parse_osm.py

```diff
@@ -261,7 +264,9 @@ def create_poi(self, tags, osmid, lat_lng, osm_type, categories=[]):
                     self.tags_object = TagsObject(my_uuid, osmid, tag, value)
                     self.store_tags(self.tags_object)
 
-            self.poi_object = PoiObject(my_uuid, categories, osmid, lat_lng, osm_type)
+            address = AddressObject(lat_lng).address_request() # json.dumps()
```

Comment:
Before the parser commences the config has to be evaluated once in terms of:
- should addresses be stored?
- if they should be requested: from which service?